xml/System/Int16.xml

```diff
@@ -1065,10 +1064,44 @@
         <Parameter Name="value" Type="System.Object" Index="0" FrameworkAlternate="dotnet-uwp-10.0;netcore-1.0;netcore-1.1;netstandard-1.0;netstandard-1.1;netstandard-1.2;netstandard-1.3;netstandard-1.4;netstandard-1.5;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">An object to compare, or <see langword="null" />.</param>
+        <summary>Compares this instance to a specified object and returns an integer that indicates whether the value of this instance is less than, equal to, or greater than the value of the object.</summary>
+        <returns>A signed number indicating the relative values of this instance and <paramref name="value" />.
+          <list type="table">
+            <listheader>
+              <term>Return Value</term>
+              <description>Description</description>
+            </listheader>
+            <item>
+              <term>Less than zero</term>
+              <description>This instance is less than <paramref name="value" />.</description>
+            </item>
+            <item>
+              <term>Zero</term>
+              <description>This instance is equal to <paramref name="value" />.</description>
```

Comment:
On the interface, we've described this as follows:

> This current instance occurs in the same position in the sort order as `value`.

For all the integral types, those two descriptions mean the same. I'll approve this, and you can decide if you want to use the same consistent language. (This comment applies to all pages in this PR)

Reply:
Hey @BillWagner would you mind elaborating a bit more? I'm not sure where you got that description from, so I'm not understanding your suggestion here.

In case it helps: I got all this text from `<Member MemberName="CompareTo">` in this same file (line 181), and simply added the extra remark about the EII.

Reply:
@carlossanlop The language I quoted above is from the [`IComparable<T>` interface page](https://docs.microsoft.com/en-us/dotnet/api/system.icomparable-1?view=netcore-3.0#remarks), in the remarks section.

For all the types in this PR, `CompareTo` returns 0 if and only if the two instances are equal. That's not true for all types.  (Imagine ordering events by date. Two events on the same date would be in the same location in the sort order, but may not be equal.)

For these types, that distinction doesn't matter. I just want to make sure we're consistent and explicit on how we make this decision.

Reply:
Oh I see what you mean. I think the descriptions are fine for these int types, they were adapter for their specific behavior.